chapter_appendix-mathematics-for-deep-learning/statistics.md

```diff
@@ -127,17 +127,17 @@ It is intuitively clear that these two components contribute to the mean squared
 $$
 \begin{aligned}
 \mathrm{MSE} (\hat{\theta}_n, \theta) &= E[(\hat{\theta}_n - E(\hat{\theta}_n) + E(\hat{\theta}_n) - \theta)^2] \\
- &= E[(\hat{\theta}_n - E(\hat{\theta}_n))^2] + E[(E(\hat{\theta}_n) - \theta)^2] \\
+ &\approx E[(\hat{\theta}_n - E(\hat{\theta}_n))^2] + E[(E(\hat{\theta}_n) - \theta)^2] \\
```

Comment:
This is an exact equal, since the "omitted" term is zero.

Reply:
The omitted term is the irreducible error.
https://en.wikipedia.org/wiki/Bias%E2%80%93variance_tradeoff
>The bias–variance decomposition is a way of analyzing a learning algorithm's expected generalization error with respect to a particular problem as a sum of three terms, the bias, variance, and a quantity called the irreducible error, resulting from noise in the problem itself.

Reply:
You are right, it makes sense to add an irreducible error. While it might not be appropariate to add an "\approx" here (or it might confuse some of the readers how to derive the formula). 

I find an another proof, which explained the irreducible error is coming from the "\Var{\theta}".  Could you help correct the proof as how it proves in https://people.eecs.berkeley.edu/~jrs/189s19/lec/12.pdf ?

Reply:
Ok!